bindings/python/iota_wallet/account.py

```diff
@@ -10,7 +10,9 @@ def __init__(self, alias_index, handle):
     def __str__(self):
         message = {
             'cmd': 'getAccount',
-            'payload': self.alias_index,
+            'payload': {
+                'accountId': self.alias_index,
```

Comment:
Just curious, alias index? That's a tad confusing

Reply:
Bing did it like this, should I rename to accountId?

Reply:
I would yeah, another PR if you prefer